src/Gelf/Logger.php

```diff
@@ -181,7 +181,7 @@ private static function interpolate($message, array $context)
         // build a replacement array with braces around the context keys
         $replace = array();
         foreach ($context as $key => $val) {
-            $replace['{' . $key . '}'] = $val;
+            $replace['{' . $key . '}'] = is_string($val) ? $val : 'Array';
```

Comment:
Oh sorry I mixed this up with another thread, given the little amount of code there I'd say go ahead and copy it :)